dockers/docker-orchagent/orch_zmq_tables.conf.j2

```diff
@@ -1,4 +1,4 @@
-{% if DEVICE_METADATA.localhost.orch_dash_zmq_enabled == "false"  %}
+{% if DEVICE_METADATA.localhost.orch_northbond_dash_zmq_enabled != "false"  %}
```

Comment:
shouldn't this be checked against "true"? If the config is disabled, we don't want all these tables in orch_zmq_tables.conf.

Please check all 4 combinations of dash_zmq_disabled/route_zmq_disabled, dash_zmq_enabled/route_zmq_disabled, dash_zmq_disabled/route_zmq_enabled, dash_zmq_enabled/route_zmq_enabled to make sure orch_zmq_tables.conf looks correct in all cases.

Reply:
@zjswhhh , @theasianpianist , if your new tables want to have swssconfig and swsssplayer command support when ZMQ enabled, please also update those tables to orch_zmq_tables.conf.j2